tensorflow/python/framework/config.py

```diff
@@ -195,39 +195,59 @@ def get_optimizer_experimental_options():
 def set_optimizer_experimental_options(options):
   """Set experimental optimizer options.
 
-  Note that optimizations are only applied in graph mode, (within tf.function).
-  In addition, as these are experimental options, the list is subject to change.
+  **Note:** Optimizations are only applied in graph mode,
+  (within `tf.function`).
+
+  By default all of the optimizers are set to False.
```

Comment:
This is not correct, the defaults values are a bit hard to determine but they are set here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/grappler/optimizers/meta_optimizer.cc#L228. I think you need to rephrase this to note to something of the effect that in order to use a non-default option or ensure a certain optimization is used/skipped.

Reply:
Ah, thank you!